lua/nvim-tree/populate.lua

```diff
@@ -159,6 +159,10 @@ end
 local should_ignore = gen_ignore_check()
 
 function M.refresh_entries(entries, cwd, parent_node)
+  if cwd == nil then
```

Comment:
This is not right, the cwd should always be defined when calling this function. The patch should come a calling function, or a calling calling function, this will probably lead to undefined behavior.